packages/compiler/core/cli.ts

```diff
@@ -54,51 +54,51 @@ async function main() {
             type: "string",
             demandOption: true,
           })
-          .option("output-path", {
+          .options("output-path", {
             type: "string",
             default: "./cadl-output",
             describe:
               "The output path for generated artifacts.  If it does not exist, it will be created.",
           })
-          .option("option", {
+          .options("options", {
             type: "array",
             string: true,
             describe:
-              "Key/value pairs that can be passed to Cadl components.  The format is 'key=value'.  This parameter can be used multiple times to add more options.",
+              "Key/value pairs that can be passed to Cadl components.  The format is 'emitter=value'.  This parameter can be used multiple times to add more options.",
```

Comment:
`emitter = value`, where value looks like what?  Generally the emitter options are keyed options, so does this become:
`--options emitter=key=value`, is 'emitter' the keyword 'emitter', or is it the full emitter name?  Is it something like `emitter.key`=value`?  We need to update the documentation, and potentially the providerhub template if the cli is significantly changing this format